### PR TITLE
Make exclude_title a config option

### PR DESCRIPTION
**Pull request by cunidev**
Works fine here, but as my knowledge of the codebase is limited any in-depth testing is welcome.

### CODE REVIEW

File: aw_watcher_window/config.py [hunk 1]
```diff
@@ -6,10 +6,12 @@
 def load_config():
     default_client_config = ConfigParser()
     default_client_config["aw-watcher-window"] = {
-        "poll_time": "1.0",
+        "exclude_title": True,
```
Comment (johan-bjareholt) [inline aw_watcher_window/config.py]: This changes the default from false to true, the default should be false

File: aw_watcher_window/config.py [hunk 2]
```diff
@@ -6,10 +6,12 @@
...
+        "poll_time": "1.0"
     }
     default_client_config["aw-watcher-window-testing"] = {
-        "poll_time": "1.0",
+        "exclude_title": True,
```
Comment (johan-bjareholt) [inline aw_watcher_window/config.py]: This changes the default from false to true, the default should be false

File: aw_watcher_window/main.py [hunk 3]
```diff
@@ -42,7 +42,7 @@ def main():
     logger.info("aw-watcher-window started")
     sleep(1)  # wait for server to start
     with client:
-        heartbeat_loop(client, bucket_id, poll_time=args.poll_time, exclude_title=args.exclude_title)
+        heartbeat_loop(client, bucket_id, poll_time=args.poll_time, exclude_title=args.exclude_title or config.getboolean("exclude_title"))
```
Comment (johan-bjareholt) [inline aw_watcher_window/main.py]: This is incorrect, an cli argument should always take precedence over a config setting.
You should handle it in the same way as the poll-time argument, pass the config value to parse_args and only use it as the default.

**Comment by cunidev** [comment]
Sorry, I've corrected that. Feel free to test the new commits

File: aw_watcher_window/main.py [hunk 4]
```diff
@@ -42,20 +42,21 @@ def main():
...
     parser.add_argument("--testing", dest="testing", action="store_true")
-    parser.add_argument("--exclude-title", dest="exclude_title", action="store_true")
+    parser.add_argument("--exclude-title", dest="exclude_title", action="store_true", default=default_exclude_title)
     parser.add_argument("--verbose", dest="verbose", action="store_true")
     parser.add_argument("--poll-time", dest="poll_time", type=float, default=default_poll_time)
     return parser.parse_args()
 
 
 def heartbeat_loop(client, bucket_id, poll_time, exclude_title=False):
+    print(exclude_title)
```
Comment (ErikBjare) [inline aw_watcher_window/main.py]: Looks like you missed this one :)

**Comment by cunidev** [comment]
Oh, someone discovered my top secret debug tools :P

**Comment by johan-bjareholt** [comment]
;Looks good!

**Pull request merged**